README.md

```diff
@@ -101,6 +101,7 @@ const styles = StyleSheet.create({
 | renderYep           | func          | renders Yep label                     |              | () => null    |
 | renderNope          | func          | renders Nope label                    |              | () => null    |
 | onSwipeStart        | func          | function to be called when a card swipe starts                  |    | () => {} |
+| onChangeSwipeDirection | func          | function to be called when a card swipe in active status                  |    | () => {} |
```

Comment:
name of the new method should be onSwipeChangeDirection, also update description to "function to be called when a card swipe change direction"